### PR TITLE
A couple of worker fixes.

### PR DESCRIPTION
**Pull request by robertwb**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @aaltay

File: sdks/python/apache_beam/runners/worker/operation_specs.py [hunk 1]
```diff
@@ -332,16 +334,16 @@ def get_coder_from_spec(coder_spec):
     return coders.IntervalWindowCoder()
   elif coder_spec['@type'] == 'kind:global_window':
     assert ('component_encodings' not in coder_spec
-            or not coder_spec['component_encodings'])
-    return coders.GlobalWindowCoder()
+            or len(coder_spec['component_encodings'] == 0))
```
Comment (aaltay) [inline sdks/python/apache_beam/runners/worker/operation_specs.py]: I think `)` is misplaced and it should read `len(coder_spec['component_encodings']) == 0)`

**Comment by aaltay** [comment]
Do we need unit tests for these code paths? It seems like we missed some places during renaming things.

File: sdks/python/apache_beam/runners/worker/operation_specs.py [hunk 2]
```diff
@@ -306,6 +306,8 @@ def get_coder_from_spec(coder_spec):
 
   # Ignore the wrappers in these encodings.
   # TODO(silviuc): Make sure with all the renamings that names below are ok.
```
Comment (aaltay) [inline sdks/python/apache_beam/runners/worker/operation_specs.py]: Can we remove this TODO?

File: sdks/python/apache_beam/runners/worker/operation_specs.py (hunk 1 above)
Comment (robertwb) [inline sdks/python/apache_beam/runners/worker/operation_specs.py]: Reverted.

File: sdks/python/apache_beam/runners/worker/operation_specs.py (hunk 2 above)
Comment (robertwb) [inline sdks/python/apache_beam/runners/worker/operation_specs.py]: Yes. Done.

**Comment by robertwb** [comment]
This is partially exercised by the map_task/fn_api runners. It will hopefully soon be obsolete as we'll use the Runner API here instead.

**Comment by aaltay** [comment]
LGTM - I apologize for the late response, it dropped out of my radar after the first review.

**Pull request closed without merging**